src/pages/ceph/users/index.vue

```diff
@@ -61,28 +81,112 @@ const { data: cephUsers } = useLazyAsyncData('cephUsers', async () => {
                                     <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">
                                         Enabled
                                     </th>
-                                    <th scope="col" class="relative py-3.5 pl-3 pr-4 sm:pr-6">
-                                        <span class="sr-only">Edit</span>
+<!--                                    <th scope="col" class="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">Edit</th>-->
```

Comment:
Do we need this code? If not, let's remove commented out code as we git history should we need it back